### PR TITLE
[FICTIUM-001] Gem Bootstrap

### PR DESCRIPTION
**Pull request by holywyvern**
## Summary

This sets up the gem for future working, travis is configured, and some rake tools are added.
Also initial dependencies are part of the application.

### CODE REVIEW

File: fictium.gemspec [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+require 'fictium/version'
+
+Gem::Specification.new do |spec| # rubocop:disable Metrics/BlockLength
+  spec.name          = 'fictium'
+  spec.version       = Fictium::VERSION
+  spec.authors       = ['Ramiro Rojo']
+  spec.email         = ['ramiro.rojo@wolox.com.ar']
+
+  spec.summary       = 'A gem to generate documentation out of tests.'
+  spec.homepage      = 'https://github.com/holywyvern/fictium'
```
Comment (blacksam07) [inline fictium.gemspec]:
```suggestion
  spec.homepage      = 'https://github.com/Wolox/fictium'
```

File: fictium.gemspec [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  spec.license       = 'Apache-2.0'
+
+  spec.metadata['homepage_uri'] = spec.homepage
+  spec.metadata['source_code_uri'] = 'https://github.com/holywyvern/fictium'
+  spec.metadata['changelog_uri'] = 'https://github.com/holywyvern/fictium/CHANGELOG.md'
```
Comment (blacksam07) [inline fictium.gemspec]: Same change here
`spec.metadata['source_code_uri'] = 'https://github.com/Wolox/fictium'`
`spec.metadata['changelog_uri'] = 'https://github.com/Wolox/fictium/CHANGELOG.md'`

File: README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+# Fictium
+
+Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/fictium`. To experiment with that code, run `bin/console` for an interactive prompt.
+
+TODO: Delete this and the text above, and describe your gem
```
Comment (redwarewolf) [inline README.md]: You had one job...

File: fictium.gemspec [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  spec.files = Dir.chdir(File.expand_path(__dir__)) do
+    `git ls-files -z`.split("\x0").reject do |f|
+      f.match(%r{^(test|spec|features)/})
+    end
+  end
+  spec.bindir        = 'exe'
+  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.require_paths = ['lib']
+
+  spec.required_ruby_version = '>= 2.6.0'
```
Comment (redwarewolf) [inline fictium.gemspec]: This can be specified in the gemspec too

Comment (redwarewolf) [reply fictium.gemspec]: Along with the rails dependency

Comment (holywyvern) [reply fictium.gemspec]: Do you mean, on the `fictium/version.rb` file ? Also, rails is not required :P (only active support)

**Pull request merged**